packages/firebase_storage/firebase_storage_web/pubspec.yaml

```diff
@@ -8,24 +8,23 @@ environment:
   flutter: ">=1.20.0 <2.0.0"
 
 dependencies:
-  async: ^2.4.2
-  crypto: ^2.1.5
+  async: ^2.5.0-nullsafety.3
+  crypto: ^3.0.0-nullsafety.0
   firebase_core: ">=0.8.0-1.0.nullsafety.1 <0.8.0-2.0.nullsafety.0"
   firebase_core_web: ">=0.3.0-1.0.nullsafety.1 <0.3.0-2.0.nullsafety.0"
   firebase_storage_platform_interface: ^1.0.4
   flutter:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  http: ^0.12.2
-  js: ^0.6.1
-  meta: ^1.1.7
+  http: ^0.13.0-nullsafety.0
```

Comment:
pulled version from the repo as null safe version is not visible from pub.dev. Not sure if this is correct.